README.md

```diff
@@ -1,73 +1,70 @@
+```
 
 And then execute:
 
     $ bundle
-
-## Usage: Normalize Numbers:
-
-    response = Ringcaptcha.normalize('353 083 148 0349')
-
-    response #=> <Ringcaptcha::Response status="SUCCESS", phone="+353831480348", country="IE", area=nil, block=nil, subscriber=nil, type="MOBILE", carrier="Vodafone">
-    response.success?   #=> true
-    response.error?     #=> false
-    response.status     #=> "SUCCESS"
-    response.phone      #=> "+353831480349"
-    response.country    #=> "IE"
-    response.area       #=> nil
-    response.block      #=> nil
-    response.subscriber #=> nil
-    response.type       #=> "MOBILE"
-    response.carrier    #=> "Vodafone"
-
-## Usage: Verifying Phone Numbers:
-
-    # Step 1 - Get a captcha token
-
-    locale = "de"
-    response = Ringcaptcha.captcha(locale)
-  
-    response.token # => "31b0332942cc4e274118098f95cba64f8eb413fa"
-
-    # Step 2 - Send the PIN code
+    
+## Setup
+In order to properly use the Ringcaptcha gem, it must be initialized by supplying your Ringcaptcha API key, as seen in the following code sample (which you could place in config/initializers/ringcaptcha.rb, if you are using rails):
+
+```ruby
+Ringcaptcha.api_key = '12345678901234567890' # your actual key will obviously differ from this
+```
+
+## Application keys
+All calls using the gem are performed on behalf of a specific application registered with Ringcaptcha. For this reason, all calls receive an application key as their first argument. While we will denote this application key as `app_key` in all the following example, your actual application key will, of course, be different.
+
+## Usage
+###Normalize phone numbers:
+
+```ruby
+response = Ringcaptcha.normalize('app_key', '353 083 148 0349')
+
+response.success?   #=> true
+response.error?     #=> false
+response.status     #=> "SUCCESS"
+response.phone      #=> "+353831480349"
+response.country    #=> "IE"
+response.area       #=> nil
+response.block      #=> nil
+response.subscriber #=> nil
+response.type       #=> "MOBILE"
+response.carrier    #=> "Vodafone"
+```
+
+###Verify a phone number:
+
+```ruby
+# Step 1 - Send a PIN code to the phone number
  
-    token = response.token
-    phone = "+353831480349"
-    service = "sms"
-    Ringcaptcha.code(token, phone, service)
-
-    # Step 3 - Verify the PIN code
-
-    code = "1234"
-    response = Ringcaptcha.verify(token, code)
+phone = "+353831480349"
+service = "sms"
+code_response = Ringcaptcha.code('app_key', token, phone, service)
 
-    response #=> #<Ringcaptcha::Response status="SUCCESS",id="2381555c031619e61b3f81af30445b27a87ae97a", phone="+353831480349", geolocation=1, phone_type="MOBILE", carrier="Vodafone", threat_level="LOW">
+# Step 2 - Verify the PIN code
 
-## Sending different app_keys (Multiple apps account)
+code = "1234" # You should request this from the PIN code recipient
+verification_response = Ringcaptcha.verify('app_key', code_response.token, code)
```

Comment:
this is no longer compatible with the previous version, maybe it's best if you keep your own branch instead of merging into mine?